app/Http/Controllers/Frontend/ExamController.php

```diff
@@ -115,6 +115,9 @@ public function storeReading(Request $request, $examId)
                 $userAnswer->is_correct = (!empty($requestQuestion['answers'][$i]['correct']) ? $requestQuestion['answers'][$i]['correct'] :0);
                 $userExam->userAnswers()->save($userAnswer);
             }
+            $exam = Exam::findOrFail($examId);
+            $exam->count_test = $exam->count_test+1;
```

Comment:
use `$exam->increment();`
https://laravel.com/docs/5.4/queries#increment-and-decrement